projects/igniteui-angular/src/lib/grids/grid/grid.search.spec.ts

```diff
@@ -687,6 +688,45 @@ describe('IgxGrid - search API #grid - ', () => {
             expect(highlights.length).toBe(1);
             expect(row.inEditMode).toBe(false);
         });
+
+        it('Should keep edit mode when tabbing, after search is applied', () => {
+            grid.primaryKey = 'ID';
+            grid.getColumnByName('Name').editable = true;
+            grid.getColumnByName('JobTitle').editable = true;
+            grid.cdr.detectChanges();
+            fix.detectChanges();
+            const cell = grid.getCellByColumn(1, 'Name');
+            const caseyCell = grid.getCellByColumn(0, 'Name');
+
+            grid.findNext('Casey');
+            grid.cdr.detectChanges();
+            fix.detectChanges();
+            let highlights = caseyCell.nativeElement.querySelectorAll(HIGHLIGHT_CSS_CLASS);
+            expect(highlights.length).toBe(1);
+
+            UIInteractions.simulateDoubleClickAndSelectEvent(cell.nativeElement);
+            fix.detectChanges();
+
+            expect(cell.editMode).toBeTruthy();
+            highlights = caseyCell.nativeElement.querySelectorAll(HIGHLIGHT_CSS_CLASS);
+            expect(highlights.length).toBe(1);
+
+            let cellInput = null;
+            cellInput = cell.nativeElement.querySelector('[igxinput]');
+            cellInput.value = 'newCellValue';
```

Comment:
You can use the method UIIteractions.setInputElementValue to send the input value